tests/time_tests/test_runner/utils.py

```diff
@@ -76,7 +76,7 @@ def filter_timetest_result(stats: dict):
         iqr, q1, q3 = calculate_iqr(time_results)
         cut_off = iqr * IQR_CUTOFF
         upd_time_results = [x for x in time_results if (q1 - cut_off < x < q3 + cut_off)]
-        filtered_stats.update({step_name: upd_time_results})
+        filtered_stats.update({step_name: upd_time_results if upd_time_results else time_results})
```

Comment:
Isn't it a way to hide an issue with invalid cutoff? I thought it is always return something if array length is > 3

Reply:
It's required for cases when array length < 2 - i.e. when we need to run exe with only 1 iteration to prepare caches. The same was done for stdev: https://github.com/openvinotoolkit/openvino/blob/master/tests/time_tests/scripts/run_timetest.py#L61